src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const parthner = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
There is a typo in the variable name `parthner`. It should be `partner` to improve code readability.